index.js

```diff
@@ -582,7 +581,14 @@ module.exports = function(app) {
       defaultId = discovered.src
       description = `Found a ${discovered.productName} with src ${discovered.src}`
     }
-   
+
+
+    if ( os.platform() == 'darwin' )
+       defaultAudioPlayer = 'afplay'
```

Comment:
Please define this var 'let defaultAudioPlayer'

Otherwise it will be at the global scope.

Reply:
Hi Scott, following up some open items. Is the implementation acceptable with "let defaultAudioPlayer" as in my changed PR?

Reply:
For existing installs, we need to set command to the default if it's not in the config.

Reply:
If I see it correctly, the current implementation has "omxplayer" hardcoded as player, except on darwin, where it is set to "afplay".

I think the implementation in the PR is doing the same via
    let defaultAudioPlayer = 'omxplayer'
    if ( os.platform() == 'darwin' )
       defaultAudioPlayer = 'afplay'

Regarding your comment: If not configured via the new option, the default is the same as before. See lines 586 where "defaultAudioPlayer" is defined and line 644 where the default value for the option "alarmAudioPlayer" is set to "defaultAudioPlayer" as defined in line 586.

Or am I missing / misunderstanding something?